theme.config.tsx

```diff
@@ -1,8 +1,13 @@
 import React from "react";
 import { DocsThemeConfig } from "nextra-theme-docs";
+import Decoder from "@components/decoder";
+import HeadComponent from "@components/head";
 
 const config: DocsThemeConfig = {
   logo: <img src="/logo.svg" style={{ height: "60px" }} />,
+  navbar: {
+    extraContent: <Decoder />,
+  },
```

Comment:
Don't add it here in the navbar! I propose we recreate a page named Encoding Playground in the Stark naming System. 

On this page, you add an input with two modes: 
1/ Encoding fricoben ==> 1499554868251
2/ Decoding 1499554868251 ==> fricoben

You need to be able to copy-paste the result of the encoding/decoding easily and get a feedback that it has been copy pasted.